tests/unit/tox_faster/main_test.py

```diff
@@ -1,6 +1,20 @@
-from tox_faster.main import hello_world
+from unittest.mock import sentinel
 
+import pytest
 
-class TestHelloWorld:
-    def test_it(self):
-        assert hello_world() == "Hello, world!"
+from tox_faster import main
+
+
+@pytest.mark.parametrize(
+    "environ,expected",
+    [
+        # In dev there's no CI envvar and tox-faster disables the envreport.
+        ({}, []),
+        # On GitHub Actions CI is set to "true" and tox-faster doesn't disable the envreport.
+        ({"CI": "true"}, None),
+    ],
+)
+def test_tox_runenvreport(mocker, environ, expected):
+    mocker.patch.dict(main.environ, environ)
```

Comment:
Does this take care of all the clean up for us? If so :+1:, this looks really handy.

Reply:
Yeah, it cleans up apparently. It doesn't autospec automatically like our `patch` fixture does though, you have to pass `autospec=True` (or in this case not since we're using `patch.dict`)